app/controllers/compute_attributes_controller.rb

```diff
@@ -21,8 +21,14 @@ def edit
 
   def update
     @set = ComputeAttribute.find(params[:id])
+
+    path = if params[:origin] == "compute_profiles"
```

Comment:
Note that the parameter will cause strong params to error out once we start enforcing it.
Perhaps it would be better to set the redirect_to on the `session` in the new/edit action and pull it out in the update/create actions instead of passing a parameter around?

Reply:
+1, agree on the strong params.